timesketch/lib/analyzers/sigma_tagger.py

```diff
@@ -69,14 +77,22 @@ def run(self):
                     rule.get('es_query'), rule.get('file_name'),
                     tag_list=rule.get('tags'))
                 tags_applied[rule.get('file_name')] += tagged_events_counter
+                time.sleep(0.5)
+                if sigma_rule_counter % 10 == 0:
+                    logger.debug('Rule {0:d}/{1:d}'.format(
+                        sigma_rule_counter,len(sigma_rules)))
             except elasticsearch.TransportError as e:
                 logger.error(
                     'Timeout executing search for {0:s}: '
                     '{1!s} waiting for 10 seconds'.format(
                         rule.get('file_name'), e), exc_info=True)
                 # this is caused by to many ES queries in short time range
                 # thus waiting for 10 seconds before sending the next one.
-                time.sleep(10)
+                time.sleep(15)
```

Comment:
same with this sleep here (as above)

Reply:
would you rather have that a config variable to be set and then default values here if no config variable is set?

Reply:
at least I would like this number, since it is a constant to be defined as such, and have a comment explaining the reasoning for this, and I'm really curious in why there is a sleep here, what is the actual reason for it?